.eslintrc.js

```diff
@@ -5,12 +5,22 @@ module.exports = {
     "prettier"
   ],
   rules: {
-    "no-var": 0,
-    "max-len": 0,
+    "max-len": "off",
+    "strict": "error",
     "prettier/prettier": "error",
   },
   env: {
     node: true,
-    mocha: true
-  }
+  },
+  parserOptions: {
+    sourceType: "script",
```

Comment:
Needed to override https://github.com/babel/eslint-config-babel/blob/master/index.js#L7 for the `strict` rule